projects/kit/components/tabs/tabs-with-more/tabs-with-more.component.ts

```diff
@@ -129,6 +144,16 @@ export class TuiTabsWithMoreComponent implements AfterViewInit {
         return Math.min(this.itemsLimit - offset, this.maxIndex);
     }
 
+    @tuiPure
+    isOverflown(index: number): boolean {
+        return index !== this.activeItemIndex || !this.options.exposeActive;
+    }
+
+    @tuiPure
```

Comment:
tuiPure is useless (methods are called from ngFor in template)

Reply:
fixed